README_maintainers.md

```diff
@@ -642,209 +624,261 @@ These commands for `-c` mean:
+| `-grey`         | These tests are currently listed as "grey", which means that they are known to be unstable or broken. These tests will not be executed by the testing framework if the option `--skipGrey` is given. If `--onlyGrey` option is given then non-"grey" tests are skipped. See `tests/Greylist.txt` for up-to-date information about greylisted tests. Please help to keep this file up to date.
 
-```
-git clone https://github.com/m0ppers/mesos-cluster
-cd mesos-cluster
-mkdir /tmp/mesos-cluster
-./start-cluster.sh /tmp/mesos-cluster/ --num-slaves=5 --rm --name mesos-cluster
-```
+### Javascript Framework
 
-Then save the following configuration to a local file and name it `arangodb3.json`:
+(Used in our local Jenkins and TravisCI integration; required for running cluster tests)
 
-```
-{
-  "id": "arangodb",
-  "cpus": 0.25,
-  "mem": 256.0,
-  "ports": [0, 0, 0],
-  "instances": 1,
-  "args": [
-    "framework",
-    "--framework_name=arangodb",
-    "--master=zk://172.17.0.2:2181/mesos",
-    "--zk=zk://172.17.0.2:2181/arangodb",
-    "--user=",
-    "--principal=pri",
-    "--role=arangodb",
-    "--mode=cluster",
-    "--async_replication=false",
-    "--minimal_resources_agent=mem(*):512;cpus(*):0.25;disk(*):512",
-    "--minimal_resources_dbserver=mem(*):1024;cpus(*):0.25;disk(*):1024",
-    "--minimal_resources_secondary=mem(*):1024;cpus(*):0.25;disk(*):1024",
-    "--minimal_resources_coordinator=mem(*):1024;cpus(*):0.25;disk(*):1024",
-    "--nr_agents=3",
-    "--nr_dbservers=2",
-    "--nr_coordinators=2",
-    "--failover_timeout=86400",
-    "--arangodb_privileged_image=false",
-    "--arangodb_force_pull_image=true",
-    "--arangodb_image=arangodb/arangodb-mesos:3.0",
-    "--secondaries_with_dbservers=false",
-    "--coordinators_with_dbservers=false"
-  ],
-  "container": {
-    "type": "DOCKER",
-    "docker": {
-      "image": "arangodb/arangodb-mesos-framework:3.0",
-      "network": "HOST"
-    }
-  },
-  "healthChecks": [
-    {
-      "protocol": "HTTP",
-      "path": "/framework/v1/health.json",
-      "gracePeriodSeconds": 3,
-      "intervalSeconds": 10,
-      "portIndex": 0,
-      "timeoutSeconds": 10,
-      "maxConsecutiveFailures": 0
-    }
-  ]
+Since several testing technologies are utilized, and different ArangoDB
+startup options may be required (even different compilation options may be
+required) the framework is split into testsuites.
+
+Get a list of the available testsuites and options by invoking: 
+
+    ./scripts/unittest
+
+To locate the suite(s) associated with a specific test file use: 
+
+    ./scripts/unittest find --test tests/js/common/shell/shell-aqlfunctions.js
+
+Run all suite(s) associated with a specific test file:
+
+    ./scripts/unittest auto --test tests/js/common/shell/shell-aqlfunctions.js
+
+Run all tests:
+
+    scripts/unittest all
+
+`scripts/unittest` is only a wrapper for the most part, the backend
+functionality lives in `js/client/modules/@arangodb/testing.js`.
```

Comment:
, process-utils.js, test-utils.js. Testsuites have their own files in `js/client/modules/@arangodb/testsuites`.

Reply:
Like this?

```suggestion
functionality lives in `js/client/modules/@arangodb/` (`testing.js`, `process-utils.js`, `test-utils.js`).
The actual testsuites are located in the `testsuites` subfolder.
```